packages/components/src/components/contract-card/contract-card-items/sass/contract-card-dialog.scss

```diff
@@ -36,9 +44,7 @@
             z-index: 2;
             color: inherit;
 
-            &-symbol {
-                font-size: 1.4rem;
-                line-height: 1.5;
```

Comment:
I'm replacing these repeatedly used font CSS styles with Text component in `Input` as a better practice. QA checklist will include all occurrences.